docs/contributor_guide/index.rst

```diff
@@ -18,3 +18,69 @@ The contribution guide explains how to structure your contributions. Please
    contributing_code
    contributing_example_notebooks
    fairlearn_proposals
+
+.. _how_to_talk_about_fairness:
+
+How to talk and/or write about fairness in AI
+---------------------------------------------
+
+- Be clear that there is no single definition of fairness that will apply
+  equally well to all AI systems.
+- Be clear that any quantitative definition of fairness will omit aspects of
+  fairness (as a societal concept) that cannot be quantified (e.g., justice,
+  due process, remedying historical societal injustices).
+- Be clear that given the many complex sources of unfairness, it is not
+  possible to fully *debias* a system or to guarantee fairness; the goal is to
+  detect and to mitigate fairness-related harms as much as possible.
+- For this reason, don’t use words like *debias*, *unbiased*, *solve* – they
+  set up unrealistic expectations; use words like *mitigate*, *address*,
+  *prioritize* instead (words like *detect*, *identify*, *assess* are fine).
+- Be clear that AI systems (and technology in general) are never *neutral* –
+  all AI systems necessarily reflect the assumptions, priorities, and values
+  of the people involved in their development and deployment.
+- Be clear that prioritizing fairness in AI systems often means making
+  tradeoffs based on competing priorities, there are seldom clear-cut answers
+  (this is why using the word *solve* is seldom appropriate).
+- Be clear that prioritizing fairness in AI systems is a sociotechnical
+  challenge – it is not something that can be accomplished via purely
+  technical methods (or purely social methods, for that matter).
+- Be clear that there is no software tool that will *solve* fairness in all AI
+  systems – this is not to say that software tools don’t have a role to play,
+  but they will be precise, targeted, and only part of the picture.
+- Be clear that even with precise, targeted tools, it’s easy to overlook
+  things, especially things that are difficult to quantify – software tools
+  must be supplemented with other resources and processes.
+- There are many reasons why AI systems can behave unfairly, not just societal
+  biases. Also *bias* is ambiguous and means different things to different
+  communities – e.g., statistical bias vs. societal biases. For this reason,
+  talk about *fairness issues* or *fairness-related harms* rather than *bias*,
+  unless you are very specifically referring to societal biases (or
+  statistical bias or some other definition of bias).
+- Better yet, be specific about the type of fairness-related harm – is this a
+  harm of allocation, quality of service, stereotyping, denigration, or over-
+  or under-representation? (But be clear that different types of
```

Comment:
Suggest getting rid of the parentheses, and just have 'or more than one of these' as the last element in the list.

Reply:
Not sure about what you meant with the latter part of your suggestion, but I've adjusted the parenthesis.

Reply:
"Better yet, be specific .... stereotyping, denigration, over/under representation, or any combination of these."

Reply:
This has all been rewritten now. Resolving the old comments because they refer to a completely different version. Let me know if you think this is better. I can always go back to the original.